draft-birkholz-cose-tsa-tst-header-parameter.md

```diff
@@ -161,6 +161,14 @@ Please review the Security Considerations section in {{-TSA}}; these considerati
 
 Also review the Security Considerations section in {{-COSE}}; these considerations apply to this document as well, especially the need for implementations to protect private key material.
 
+We assume an attacker who has the ability to manipulate the clocks on the COSE signer's clock and its relying parties, but not the clock of the TSA.
```

Comment:
```suggestion
Attackers could achieve the ability to manipulate the local clocks used by the COSE signer or local clocks of its relying parties, but not the clock of the TSA.
```

Reply:
the spirit of this suggestion has been captured in [e75174a](https://github.com/ietf-scitt/draft-birkholz-cose-tsa-tst-header-parameter/pull/24/commits/e75174a9281914f13bbd31b6287ad4e029991169)